api/config/rolepermission.php

```diff
@@ -0,0 +1,521 @@
+<?php
+
```

Comment:
Could you add a comment explaining this is used by RolePermissionSeeder, and a link to the Laratrust Documentation this is based on?

Reply:
Add docs in [update config with docs and new keys](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5560/commits/68fe9b46e66ce421900acf0b5a55799886c1bf77)